src/components/molecules/EditRoomForm/EditRoomForm.scss

```
@@ -10,6 +10,17 @@
     padding: $spacing--xl;
   }
 
+  &__dropdown {
+    appearance: none;
+    border-radius: $border-radius--xl;
+    color: $white;
+    background-color: opaque-white(0.12);
+    padding: $spacing--md $spacing--xl $spacing--md $spacing--md;
+    text-align: center;
+    width: 100%;
+    margin-bottom: $spacing--xl;
+  }
```

A way to fix the drop down list background and text to not having the same color.

```suggestion
    &:hover {
      background-color: opaque-white(0.16);
    }

    option {
      color: $white;
      background-color: $secondary--dark;
    }
  }
```

---

fixed in 2ab4e9997f9341a3a86c13ea0a77a77b55a15131